src/migration-scripts/interfaces/29-to-30

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2021-2023 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Deletes Wireguard peers if they have the same public key as the router has.
+import sys
+from vyos.configtree import ConfigTree
+from vyos.validate import is_wireguard_key_pair
+
+if __name__ == '__main__':
+    if (len(sys.argv) < 1):
```

Comment:
Should the len be 2?

Reply:
Why 24?

Here is what it does: It counts the number of command line arguments. Command line argument 0 corresponds to the binary name (same in C/C++). Command line argument 1 would be the config file to migrate.

Example:

```python
#!/usr/bin/env python3
import sys
print(sys.argv)
```

```bash
$ ./foo.py
['./foo.py']


$ ./foo.py bar
['./foo.py', 'bar']
```

Reply:
Maybe I'm confusing, but len by default will be 1 without any args
With one config args, the len will be 2, isn't it?
```
root@r14:/home/vyos# cat argtest.py 
#!/usr/bin/env python3

#!/usr/bin/env python3
import sys
print(sys.argv)

print(len(sys.argv))
root@r14:/home/vyos# 
root@r14:/home/vyos# ./argtest.py 
['./argtest.py']
1
root@r14:/home/vyos# 
```

Reply:
I have tested. It really should be 2. Changed.